diff -Naur -p1 gnome-control-center-3.20.1.orig/panels/background/bg-wallpapers-source.c gnome-control-center-3.20.1/panels/background/bg-wallpapers-source.c
--- gnome-control-center-3.20.1.orig/panels/background/bg-wallpapers-source.c	2015-03-23 17:55:50.000000000 +0700
+++ gnome-control-center-3.20.1/panels/background/bg-wallpapers-source.c	2016-06-13 20:04:05.480067773 +0700
@@ -116,3 +116,3 @@ load_default_bg (BgWallpapersSource *sel
 				 "gnome-background-properties",
-				 "adwaita.xml",
+				 "blankon-wallpapers.xml",
 				 NULL);
